wys/api/python/wys_api.py

```diff
@@ -110,23 +110,23 @@ def get_statistics_hourly(location, start_date, hr, api_key):
                         ':00/to/'+str(hr).zfill(2)+':59/speed_units/0', 
                         headers=headers)
     # Error handling
+    try:
+        res = response.json()
```

Comment:
I think this should start with a `response.raise_for_status()` and follow some of the structure from the `here_api.py`

https://github.com/CityofToronto/bdit_data-sources/pull/497/files